apps/artifactory/artifactory-helm/artifactory.yaml

```diff
@@ -43,3 +43,585 @@ spec:
         className: traefik
         annotations:
           traefik.ingress.kubernetes.io/router.tls: "true"
+      configMapName: artifactory.config.import.xml
+      configMaps: |
+        artifactory.config.import.xml: |
```

Comment:
Anyway to source this as a file or is that a pain?

Reply:
have isolated it to its own patch file